src/components/cell/Cell.js

```diff
@@ -1,10 +1,11 @@
-import React, { memo } from 'react';
+import React, { memo, useContext } from 'react';
 import PropTypes from 'prop-types';
 import { Typography } from '@material-ui/core';
 import isEqual from 'lodash.isequal';
 import Autocomplete from '@material-ui/lab/Autocomplete';
 import TextField from '@material-ui/core/TextField';
 
+import { MarkdownContext } from 'markdown-translatable';
 import { BlockEditable } from 'markdown-translatable';
```

Comment:
Let's combine the imports.
```suggestion
import { BlockEditable, MarkdownContext } from 'markdown-translatable';
```